docs/contribute.md

```diff
@@ -1,328 +1,141 @@
-only `elasticsearch` and `swift` services are accessible as docker containers
-(see `docker-compose.yml` services).
+### 5. Closing issues
 
-To configure those backends, we provide default parameters in the `.env.dist`
-template, you can copy/paste them in your `.env` file (and uncomment them so
-that they are properly injected in running containers).
+Issues will be closed by project maintainers once they are resolved or deemed invalid. You can always reopen an issue if you believe it hasn't been adequately addressed.
 
-> In order to run the Elasticsearch backend locally on GNU/Linux operating
-> systems, ensure that your virtual memory limits are not too low and increase
-> them (temporally) if needed by typing this command from your terminal (as
-> `root` or using `sudo`):
->
-> `sysctl -w vm.max_map_count=262144`
->
-> Reference:
-> https://www.elastic.co/guide/en/elasticsearch/reference/master/vm-max-map-count.html
+## Discussions
 
-Once configured, start available backends using:
+Discussions in the Ralph repository are a place for open-ended conversations, questions, and general community interactions. Here's how to effectively use discussions:
 
-```bash
-$ make run-[BACKEND]
-```
+### 1. Creating a Discussion
 
-Substitute `[BACKEND]` by the backend name, _e.g._ `es` for Elasticsearch or
-`swift` for OpenStack Swift:
+- Use a clear and concise title that summarizes the topic.
+- In the description, provide context and details regarding the discussion.
+- Use labels to categorize the discussion (e.g., "question," "general discussion," "announcements," etc.).
 
-```bash
-# Start Elasticsearch local backend
-$ make run-es
-# Start Swift local backend
-$ make run-swift
-# Start all local backends
-$ make run-all
-```
+### 2. Participating in Discussions
 
-Now that you have started at least the `elasticsearch` and `swift` backends,
-it's time to play with them:
+- Engage in conversations respectfully, respecting others' opinions.
+- Avoid spamming or making off-topic comments.
+- Help answer questions when you can.
 
-```bash
-# Store a JSON file in the Swift backend
-$ echo '{"id": 1, "foo": "bar"}' | \
-    ./bin/ralph write -b swift -f foo.json
+### 3. Announcements
 
-# Check that we have created a new JSON file in the Swift backend
-$ bin/ralph list -b swift
-foo.json
+Project maintainers may use discussions to make announcements or share important updates.
+Be sure to follow these discussions to stay informed about the project's progress.
 
-# Read the content of the JSON file and index it in Elasticsearch
-$ bin/ralph read -b swift foo.json | \
-    bin/ralph write -b es
+### 4. Code of conduct 
 
-# Check that we have properly indexed the JSON file in Elasticsearch
-$ bin/ralph read -b es
-{"id": 1, "foo": "bar"}
-```
+Remember to maintain a positive and friendly atmosphere in discussions. 
+Use discussions as a space for productive interactions and community building.
```

Comment:
This is the 3rd time people are reminded to be friendly or respecful. Maybe this can be said once in a general "Code of conduct" section meant to be applied to all interactions ?